tsconfig.release.json

```diff
@@ -1,18 +1,12 @@
 {
   "compilerOptions": {
     "declaration": true,
-    "lib": ["es6"],
+    "lib": ["es2017"],
     "module": "commonjs",
     "noImplicitAny": false,
     "outDir": "lib",
     "stripInternal": true,
-    "target": "es6",
-    "typeRoots": [
```

Comment:
Just noticed this - do we not need this field anymore? This seems like it was limiting us to only pick up types in `node_modules` and not deeper down the tree.

Reply:
It is indeed more strict, although it pulls types _more shallow_ in the tree, not _deeper_. So if there would be an enclosing directory that also have `node_modules`, then `@types` from this directory would be pulled too. I'm all for strictness, therefore I added it back in d9d9fcb48bcd7e8fef1ecd87a01384e332d7dc97.

Reply:
Thanks for pointing this out, never thought about it like this.

Reply:
This looks good, thanks for adding back in.